src/common/messaging.ts

```diff
@@ -42,6 +42,12 @@ export interface SessionContext {
     sessionId?: string;
     canRead: boolean;
     canWrite: boolean;
+    stopped?: boolean;
+}
+
+export interface ViewState {
+    active: boolean;
+    visible: boolean;
```

Comment:
This interface is passed around, but I don't see that its values are ever used?

Reply:
Yeah, good catch! It is a remnant of the code we had for the focus change, i.e., the code that is currently commented:

        // activate the code below if you want to refresh the memory when the view becomes active (focussed)
        // const viewStateChange: Change<ViewState> = { from: from.viewState, to: current.viewState };
        // if (hasChangedTo(viewStateChange, 'active', true)) {
        //     this.fetchMemory();
        // }

I'll remove the view state related code and remove the commented code as well. We'll always have the PR to look up any history, it does not need to be in the submitted change.